spec/clj/tic_tac_toe_clojure/core_spec.clj

```diff
@@ -253,16 +341,16 @@
 
   (it "checks if three symbols are aligned"
     (should= true
-      (three-aligned? ["X" "_" "O"
+      (n-aligned? ["X" "_" "O"
```

Comment:
Is there a way to abstract `n` to get something like
`(mark-aligned? the-board the-mark)`